src/pages/Debug/DebugDetails.tsx

```diff
@@ -236,7 +237,10 @@ function DebugDetails({data, children, onSave, onDelete, validate}: DebugDetails
                         text={translate('common.delete')}
                         onPress={() => {
                             onDelete();
-                            Navigation.goBack();
+                            // Dismiss the current report screen and replace it with Concierge Chat
```

Comment:
@pac-guerreiro Why should we do that?

Reply:
@DylanDylann This is a workaround I found to prevent the app from fetching the report data again, it's based on this:

<img width="1172" alt="Screenshot 2024-10-29 at 10 21 22" src="https://github.com/user-attachments/assets/03cea8bc-aa07-417d-9eba-7a7fb241628a">

But now that I'm seeing this again, I think I should just use `Report.navigateToConciergeChatAndDeleteReport` instead of duplicating code